### ISSUE TITLE
Error parsing SQL: '/*!50003 DROP FUNCTION IF EXISTS `DAY_NAME_FROM_NUMER` */'

### ISSUE THREAD

**Issue by nmaquet** [zendesk/maxwell#254]
Maxwell crashed with the following error today (1.0-PRE1):

```
04:31:33,645 ERROR MysqlParserListener - (statement DROP FUNCTION IF EXISTS `DAY_NAME_FROM_NUMER`)
04:31:33,646 ERROR SchemaChange - Error parsing SQL: '/*!50003 DROP FUNCTION IF EXISTS `DAY_NAME_FROM_NUMER` */'
com.zendesk.maxwell.schema.ddl.MaxwellSQLSyntaxError: DROP
        at com.zendesk.maxwell.schema.ddl.MysqlParserListener.visitErrorNode(MysqlParserListener.java:96)
        at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:41)
        at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
        at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
        at com.zendesk.maxwell.schema.ddl.SchemaChange.parseSQL(SchemaChange.java:73)
        at com.zendesk.maxwell.schema.ddl.SchemaChange.parse(SchemaChange.java:85)
        at com.zendesk.maxwell.MaxwellReplicator.processQueryEvent(MaxwellReplicator.java:310)
        at com.zendesk.maxwell.MaxwellReplicator.getRow(MaxwellReplicator.java:290)
        at com.zendesk.maxwell.MaxwellReplicator.work(MaxwellReplicator.java:101)
        at com.zendesk.maxwell.RunLoopProcess.runLoop(RunLoopProcess.java:31)
        at com.zendesk.maxwell.Maxwell.run(Maxwell.java:100)
        at com.zendesk.maxwell.Maxwell.main(Maxwell.java:106)
04:31:33,647 INFO  SchemaPosition - Storing final position: null
```

**Comment by osheroff** [comment]
k.  I think the solution is to use a cheap regex to unwrap sql from comments before checking against the blacklist.  Can hopefully look today.

**Comment by osheroff** [comment]
fixed in 1.0.0-PRE2
